src/dex-ui/services/MirrorNodeService/MirrorNodeService.ts

```diff
@@ -218,39 +252,45 @@ function createMirrorNodeService() {
         limit: 100,
       },
     });
-    const proposals: MirrorNodeDecodedProposalEvent[] = response.data.logs
-      .flatMap((proposalEventLog: MirrorNodeProposalEventLog) => {
-        const proposalCreatedEvent = decodeEvent(
-          "ProposalCreated",
-          proposalEventLog.data,
-          proposalEventLog.topics.slice(1)
-        );
-        /* TODO: Fix claim, executed, and canceled event logs. This may need to be addressed in the
-        contracts code.
-        const godTokenClaimedEvent = decodeEvent(
-          "GodTokenClaimed",
-          proposalEventLog.data,
-          proposalEventLog.topics.slice(1)
-        );
-        let areGODTokensClaimed = false;
-        if (Boolean(godTokenClaimedEvent?.proposalId) && Boolean(proposalCreatedEvent?.proposalId))
-          areGODTokensClaimed = godTokenClaimedEvent?.proposalId === proposalCreatedEvent?.proposalId;
 
-        const proposalExecutedEvent = decodeEvent(
-          "ProposalExecuted",
-          proposalEventLog.data,
-          proposalEventLog.topics.slice(1)
-        );
-        const proposalCanceledEvent = decodeEvent(
-          "ProposalCanceled",
-          proposalEventLog.data,
-          proposalEventLog.topics.slice(1)
-        ); 
-        console.log(proposalCreatedEvent, godTokenClaimedEvent, proposalExecutedEvent, proposalCanceledEvent);
-        */
-        return [proposalCreatedEvent ? { ...proposalCreatedEvent, contractId, type: proposalType } : undefined];
-      })
-      .filter((proposal: MirrorNodeDecodedProposalEvent | undefined) => proposal !== undefined);
+    const allEvents = decodeLog(loadGovernorAbiEventSignatures(), response.data.logs);
```

Comment:
Since the `abi` is a constant value, we can call `loadGovernorAbiEventSignatures` to produce the `govenorAbiSingatureMap` value in a separate constants file and import that value in this file.